cypress/e2e/local/ineligibleApplicantSucessfullyRejected.cy.ts

```diff
@@ -0,0 +1,118 @@
+import { faker } from '@faker-js/faker/locale/en_GB';
+import HomePage from '../../pages/home';
+import SignInPage from '../../pages/signIn';
+import VerifyPage from '../../pages/verify';
+import StartPage from '../../pages/start';
+import {
+  generateEmailAddress,
+  getRandomGender,
+  TitleEnum,
+} from '../../../testUtils/personHelper';
+import AgreeTermsPage from '../../pages/agreeTerms';
+import ApplyHouseholdPage from '../../pages/household';
+import ApplyExpectPage from '../../pages/apply/expect';
+import RejectionPage from '../../pages/rejection';
+
+import ApplyResidentSummaryPage from '../../pages/apply/[resident]/summary';
+
+// user is under 18
+const birthDate = faker.date.birthdate({ mode: 'age', min: 15, max: 17 });
+const title = faker.helpers.enumValue(TitleEnum);
+const phoneNumber = faker.phone.number();
+const mainApplicantFirstName = faker.person.firstName();
+const mainApplicantLastName = faker.person.lastName();
+const postcode = 'A1 2BC';
+const email = generateEmailAddress();
+const applicationId = faker.string.uuid();
+const verificationCode = faker.number
+  .int({ min: 100000, max: 999999 })
+  .toString();
+
+const fillInTheSignUpForm = () => {
+  StartPage.getTitleDropdown().select(title);
+  StartPage.getFirstNameInput().type(mainApplicantFirstName);
+  StartPage.getLastNameInput().type(mainApplicantLastName);
+  StartPage.getDoBDayInput().type(birthDate.getDate().toString());
+  StartPage.getDoBMonthInput().type((birthDate.getMonth() + 1).toString());
+  StartPage.getDoBYearInput().type(birthDate.getFullYear().toString());
+  StartPage.getGenderOptions().check(getRandomGender());
+  StartPage.getNationalInsuranceNumberInput().type(
+    faker.string.alphanumeric(9)
+  );
+  StartPage.getPhoneNumberInput().type(phoneNumber);
+};
+
+const addressSearchAPIResponse = {
+  body: {
+    address: [
+      {
+        line1: 'TEST ADDRESS',
+        line2: '1 STREET',
+        line3: 'LOCAL',
+        line4: '',
+        town: 'CITY',
+        postcode: `${postcode}`,
+        UPRN: 11111111111,
+      },
+    ],
+    page_count: 1,
+    total_count: 1,
+  },
+};
+
+Cypress._.times(1, () => {
```

Comment:
If this runs only once, perhaps, we don't need this wrapper?

Reply:
It's mostly here because it's been useful as a local test to run multiple times to shake out any errors.

Reply:
Is it still needed? :thinking:

Reply:
If not, then it's better to remove as it would be comparable to dead code.